gnomad_qc/v4/annotations/generate_freq_genomes.py

```diff
@@ -839,15 +870,18 @@ def compute_an_by_group_membership(
             sex_karyotype=vmt.meta.sex_imputation.sex_karyotype,
             group_membership=group_membership_ht[vmt.col_key].group_membership,
         )
-        .select_entries("AD", "DP", "GT", "GQ")
+        .select_entries("LA", "LAD", "DP", "LGT", "GQ")
         .select_rows()
     )
+    vmt = hl.experimental.sparse_split_multi(vmt, filter_changed_loci=True)
```

Comment:
I think you might need to move your quoted part here after split_multi.

Reply:
Sorry just saw this, I changed that around on my last push when doing cleanup